# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/bacon/hotfix-which.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- bacon-4.4/bacon.sh.vanilla	2021-12-27 17:12:39.993000000 +0100
+++ bacon-4.4/bacon.sh	2021-12-27 17:12:58.329000000 +0100
@@ -62,13 +62,6 @@
 # Our numerical environment is POSIX
 export LC_NUMERIC="POSIX"
 
-# Find coretools
-if [[ -z `which cat` || -z `which rm` || -z `which tr` || -z `which touch` || -z `which uname` || -z `which head` || -z `which pwd` ]]
-then
-    echo "System error: 'cat', 'rm', 'tr', 'touch', 'uname', 'head' or 'pwd' not found on this system!"
-    exit 1
-fi
-
 # Solaris
 if [[ `uname` = +(*SunOS*) ]]
 then
